src/main/java/seedu/address/model/time/TimePeriod.java

```diff
@@ -117,10 +118,10 @@ public List<HourBlock> fragmentIntoHourBlocks() {
 
     @Override
     public String toString() {
-        return String.format("[%s] %s - %s",
+        return String.format("[%s] %d - %d",
             getSchoolDay(),
-            TimeUtil.formatLocalTime(getStartTime()),
-            TimeUtil.formatLocalTime(getEndTime()));
+            getStartTime().getHourOfDay(),
+            getEndTime().getHourOfDay());
```

Comment:
`TimeUtils.formatLocalTime` gives you the AM and PM too. I think you should revert it back. I use it for debugging purposes and it is easier for me.

Reply:
The local time gives it in full HH:MM:SS:milli so it was VERY painful when I was debugging.

Reply:
Idk why you would want to display the full string.

Reply:
If you use time utils, instead of 08:00:00, I have 8 AM. Its more intuitive

Reply:
The toString method definitely needs to improve. The user doesn’t need to see anything smaller than an hour, since our application does not allow for such values. It makes our jobs as developers much more difficult when debugging.

Reply:
No, it doesn’t output as 8AM.

Reply:
It gives the full thing.

Reply:
Agreement reached to revert